### PR TITLE
changed "as usize" to "from"

### PR DESCRIPTION
**Pull request by zafarrancho**

### CODE REVIEW

File: src/base.rs [hunk 1]
```diff
@@ -65,6 +65,13 @@ pub struct Sudoku<T> {
     arr: [[T; 9]; 9],
 }
 
+impl From<Sudoku<T>> for Sudoku<U> {
+    fn from (item: Sudoku<T> -> Sudoku<U> {
+
+
+
```
Comment (nh2) [inline src/base.rs:74]: It seems some lines are missing here.

File: src/base.rs [hunk 2]
```diff
@@ -121,16 +128,16 @@ impl<T> Sudoku<T> {
     }
 
     pub fn block_for_cell<'a>(&'a self, r: Ix, c: Ix) -> impl Iterator<Item = &'a T> {
-        let r_min = (r as usize / 3) * 3;
-        let c_min = (c as usize / 3) * 3;
+        let r_min = (usize::form(r) / 3) * 3;
+        let c_min = (usize::form(c) / 3) * 3;
```
Comment (nh2) [inline src/base.rs]: Typo `form` -> `from`?

**Review by nh2 — CHANGES_REQUESTED** [review]
I added some comments

File: src/base.rs (hunk 1 above)
Comment (zafarrancho) [reply src/base.rs:74]: Done

File: src/base.rs (hunk 2 above)
Comment (zafarrancho) [reply src/base.rs]: Done

**Review by nh2 — APPROVED** [review]

**Pull request merged**